### ISSUE TITLE
Rendering broken in IE11

### ISSUE THREAD

**Issue by kuon** [mdgriffith/stylish-elephants#50]
IE11 rendering is completely broken with SE. Everything is overlapping horizontally like element has no width.

Also on the Y axis, elements are not filling containers.

**Comment by atlewee** [comment]
Its gotten a little better now with 5.0.0, but still completly broken.
I can see that my buttons/elements is exactly as wide as my paddingX.
It seems that text inside elements is not taking up any space.  
Padding adds width, but text does not.

I guess a bad but somewhat working hack would be to create your own "text" element that wraps your text inside an element with calculated width based on the length.  :)
```
text string =
    let 
        textWidth = 
            ( String.length string * pixelsPrCharacter )
    in
        el [ width textWidth ] (Element.text string)
```
Have not tried it, but should work in theory ?
Or is there any htmlAttributes we can set to work around this issue?

**Comment by mdgriffith** [comment]
Yeah, I'm still looking into this.  Edge support got a lot better, but I definitely see the issues with IE11. I released the new version of the alpha just because I didn't want to hold it up any longer.

I'm not %100 on how to fix it, but there's a lot I haven't tried.

**Comment by kuon** [comment]
@atlewee sadly, measuring text is very complicated (think kerning, asian characters, arab....), so this is not an option.

What works (I think) is to wrap the text into an element not displayed with flex. I'm going to test.

**Comment by atlewee** [comment]
I don't know if this helps to understand what is happening, but I tried to set ("min-width", "1px") on the parent.
This makes also Firefox display 100% identical to IE11.  (Elements becomes 0px wide)
